Source/Bricks/Collection/CollectionBrick.swift

```diff
@@ -15,6 +15,7 @@ public typealias RegisterBricksCollectionBrickBlock = ((_ cell: CollectionBrickC
 
 open class CollectionBrick: Brick {
     weak var dataSource: CollectionBrickCellDataSource?
+    weak public var delegate: CollectionBrickCellDelegate?
```

Comment:
A `CollectionBrickCellDelegate` should be placed on `CollectionBrickCell`, not `CollectionBrick`.

Reply:
This is the common pattern with all of our current bricks. Did we decide to change this up? I'd rather stay consistent with how things are currently done, unless we made a conscious choice to ensure all dataSources and delegates are named and places in that way.

Reply:
fair game. This is a counter-intuitive quirk of the current APIs and should definitely be revisited by the workgroup. Regardless, in some of the cases where there are delegate members of bricks, their names express the fact that they are for their cells (specifically on the Brick model). If this is just going to be named `delegate` it should be a `CollectionBrickDelegate` and if it's specifically for cells (which I still don't really buy), it should be `weak public var brickCellDelegate: CollectionBrickCellDelegate`.

Reply:
yeah i 100% agree with the names not matching up, but for whatever reason it is how things have been going and if we want to change i'd like to be formal and announced so that other PRs like this will be flagged instead of having 2 nomenclatures individually progressing on their own

Reply:
so what is the proposed change? if any?

Reply:
i'd really rather stay consistent until work group announces a change, though

Reply:
`delegate` is correct in this context if we are being consistent with the rest of brickkit.